contrib/polarity-for-haskellers/README.md

```diff
@@ -0,0 +1,115 @@
+# Polarity for Haskellers
+
+It might be a bit confusing to understand what polarity's `data` and `codata` and `def` and `codef` do differently than Haskell's data. Especially observing infinite streams is incredibly easy in Haskell.
+
+Take for example:
+
+```haskell
+data Stream a = MkStream a (Stream a)
+
+ones :: Stream Int
+ones = MkStream 1 ones
+```
+Now, we could easily just observe the 5th element by pealing off 5 `MkStream` constructors and returning the the element in the cell contained there; No need of `codata`.
```

Comment:
```suggestion
Now, we could easily just observe the 5th element by pealing off 5 `MkStream` constructors and returning the the element in the cell contained there; no need of `codata`.
```

Reply:
ah look, right next to it is another one :P thanks